src/polyhedralGravity/calculation/GravityModel.cpp

```diff
@@ -17,11 +18,12 @@ namespace polyhedralGravity {
                             "Starting to iterate over the planes...");
         GravityModelResult result{};
         result = thrust::transform_reduce(
-#ifdef DEVICE
                 thrust::device,
-#endif
-                polyhedronIterator.first, polyhedronIterator.second, [](const Array3Triplet &face) {
+                polyhedronIterator.first,
+                polyhedronIterator.second,
+                [](const Array3Triplet &face) {
                     using namespace util;
+                    using namespace detail;
```

Comment:
that is a very vague namespace name. detail of what? not sure what it does, maybe there is a bit more explicit name you could use?

Reply:
It's more of a non-formalized convention I'm using here since namespaces don't have `private` members or a similar concept. E.g., `boost` keeps it like that.
Compare: https://stackoverflow.com/questions/26546265/what-is-the-detail-namespace-commonly-used-for